### ISSUE TITLE
Failed to set schedule

### ISSUE THREAD

**Issue by joaovit1** [wkaisertexas/tiktok-uploader#127]
I am using the updated repo changes made 2 days ago and I am still having issues with video scheduling and hashtags, is anyone else having this issue?

```
Failed to set schedule: Message: element not interactable
 (Session info: chrome=123.0.6312.124)
Stacktrace:
        GetHandleVerifier [0x00B04CA3+225091]
        (No symbol) [0x00A34DF1]
        (No symbol) [0x008D9944]
        (No symbol) [0x00912667]
        (No symbol) [0x009098DE]
        (No symbol) [0x0092F55C]
        (No symbol) [0x0090930E]
        (No symbol) [0x0092F7F4]
        (No symbol) [0x00945CB0]
        (No symbol) [0x0092F2F6]
        (No symbol) [0x009079B9]
        (No symbol) [0x0090879D]
        sqlite3_dbdata_init [0x00F79A43+4064547]
        sqlite3_dbdata_init [0x00F8104A+4094762]
        sqlite3_dbdata_init [0x00F7B948+4072488]
        sqlite3_dbdata_init [0x00C7C9A9+930953]
        (No symbol) [0x00A407C4]
        (No symbol) [0x00A3ACE8]
        (No symbol) [0x00A3AE11]
        (No symbol) [0x00A2CA80]
        BaseThreadInitThunk [0x76E6FCC9+25]
        RtlGetAppContainerNamedObjectPath [0x77857C5E+286]
        RtlGetAppContainerNamedObjectPath [0x77857C2E+238]
```

**Comment by romanrsr** [comment]
same here

**Comment by developpeur-windev-php-python** [comment]
i have the same problem today after update the package

**Comment by callum-fortune** [comment]
@joaovit1 Whilst I'm not sure at this moment about the scheduling because I haven't really looked, I have addressed the hashtag issue in this PR (https://github.com/wkaisertexas/tiktok-uploader/pull/128).

**Comment by joaovit1** [comment]
The hashtags are working as a charm, thanks!

**Comment by joaovit1** [comment]
Also fixed scheduling in PR (#130), closing the issue.
